---
title: "A consensus protocol for the recovery of mercury methylation genes from metagenomes"
date: 2023-01-01
type: paper
authors:
  - "Capo, E."
  - "Peterson, B. D."
  - "Kim, M."
  - "Jones, D. S."
  - "Acinas, S. G."
  - "Amyot, M."
  - "Bertilsson, S."
  - "Björn, E."
  - "Buck, M."
  - "Cosio, C."
  - "Elias, D. A."
  - "Gilmour, C."
  - "Goñi-Urriza, M."
  - "Gu, B."
  - "Lin, H."
  - "Liu, Y.-R."
  - "McMahon, K."
  - "Moreau, J. W."
  - "Pinhassi, J."
  - "Sánchez, P."
  - "et al."
  - "Gionfriddo, C. M."
journal: "Molecular Ecology Resources"
volume: "23(1)"
pages: "190-204"
doi: "10.1111/1755-0998.13687"
link: "https://doi.org/10.1111/1755-0998.13687"
keywords:
  - "Metagenomics"
  - "Bioinformatics"
  - "Hg methylation"
  - "Hg-MATE"
  - "hgcAB genes"
  - "Marky-coco"
  - "Mercury"
abstract: "Mercury (Hg) methylation genes (hgcAB) mediate the formation of the toxic methylmercury and have been identified from diverse environments, including freshwater and marine ecosystems, Arctic permafrost, forest and paddy soils, coal-ash amended sediments, chlor-alkali plants discharges and geothermal springs. Here we present the first attempt at a standardized protocol for the detection, identification and quantification of hgc genes from metagenomes. Our Hg-cycling microorganisms in aquatic and terrestrial ecosystems (Hg-MATE) database, a catalogue of hgc genes, provides the most accurate information to date on the taxonomic identity and functional/metabolic attributes of microorganisms responsible for Hg methylation in the environment. Furthermore, we introduce 'marky-coco', a ready-to-use bioinformatic pipeline based on de novo single-metagenome assembly, for easy and accurate characterization of hgc genes from environmental samples. We compared the recovery of hgc genes from environmental metagenomes using the marky-coco pipeline with an approach based on coassembly of multiple metagenomes. Our data show similar efficiency in both approaches for most environments except those with high diversity (i.e., paddy soils) for which a coassembly approach was preferred. Finally, we discuss the definition of true hgc genes and methods to normalize hgc gene counts from metagenomes."
---
