---
title: "Unmasking the physiology of mercury detoxifying bacteria from polluted sediments"
date: 2024-03-01
type: paper
authors:
  - "Pereira-García, C."
  - "del Amo, E. H."
  - "Vigués, N."
  - "Rey-Velasco, X."
  - "Rincón-Tomás, B."
  - "Pérez-Cruz, C."
  - "Sanz-Sáez, I."
  - "Hu, H."
  - "Bertilsson, S."
  - "Pannier, A."
  - "Soltmann, U."
  - "Sánchez, P."
  - "Acinas, S. G."
  - "Bravo, A. G."
  - "Alonso-Sáez, L."
  - "Sánchez, O."
journal: "Journal of Hazardous Materials"
volume: "467"
pages: "133685"
doi: "10.1016/j.jhazmat.2024.133685"
link: "https://doi.org/10.1016/j.jhazmat.2024.133685"
keywords:
  - "Mercury bioremediation"
  - "Hg-resistant bacteria"
  - "Phylogenomics"
  - "Transcriptomics"
  - "Lipopolysaccharide metabolism"
abstract: "The microbial-driven mercury (Hg) detoxification process is mainly based on the reduction of ionic mercury to the less toxic elemental Hg. However, information on this process and, especially, on the physiology of Hg-resistant bacteria (HgRB) is still scarce. Here, we use a combination of cultivation, phenotypic and phylogenomic characterization, and time-series transcriptomics to explore the physiology of HgRB from Hg-polluted marine sediments. We isolate, and characterize 74 HgRB strains. Minimum Inhibitory Concentration (MIC) values range from 2.5 to 80 µM for ionic Hg and from 0.25 to 40 µM for MeHg. Pangenome analyses evidence an accessory genome containing key genes putatively involved in heavy metal-related processes, with mer operons in 64% of the 74 strains. Time-series transcriptomics of selected strains show an overall overexpression of the mer operon and genes for carbohydrate metabolism, lipopolysaccharide modifications, and antioxidant responses related to Hg detoxification that occur after 8 h of Hg exposure. This work reveals that HgRB use multiple strategies to protect themselves from the toxic effects of Hg and provides a valuable resource for the bioremediation of Hg-polluted environments."
---
